library/src/main/java/com/chuckerteam/chucker/internal/support/JsonConverter.kt

```diff
@@ -1,19 +1,11 @@
 package com.chuckerteam.chucker.internal.support
 
-import com.google.gson.FieldNamingPolicy
-import com.google.gson.Gson
-import com.google.gson.GsonBuilder
-import com.google.gson.internal.bind.DateTypeAdapter
-import java.util.Date
+import com.squareup.moshi.Moshi
 
 internal object JsonConverter {
```

Comment:
Imho we could nuke `JsonConverter` at all as it's exposing its implementation details. I would rather move this `Moshi` instance to the `MoshiUtil` file and have it as a top level field.

Reply:
Yes, makes sense.

Should I change the name from `JsonConverter.instance` to `moshi` as it is a `Moshi` instance? 

something like this
```
internal val moshi: Moshi by lazy {
    Moshi.Builder().build()
}
```

and usage like this
```
moshi.getHttpHeaderListJsonAdapter().toJson(headers)
```

Reply:
Yes that is definitely better.

Reply:
Done.